From 25970712c4782f50d1478f0958cb5eaab140bf53 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@redhat.com>
Date: Fri, 17 Jun 2022 15:05:54 -0500
Subject: [PATCH 4/4] Build against webkit2gtk-4.1
X-Developer-Signature: v=1; a=openpgp-sha256; l=683; i=xtexchooser@duck.com;
 h=from:subject; bh=iUsOSOSEd3c8Y86XACWHdl44YinE3TnHV5gVLwL0JJ4=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDBlWTk9u/bp8a35xo+F+kySF7+t0Odzvy6zbv8msXDZz+
 tq9G37wd5SyMIhxMciKKbIUGTZ4s+qk84suK5eFmcPKBDKEgYtTACZy/x8jw1XBQ3wHTnw/fZib
 Q+2JrYBlwJUWaZ/3D6cvdM52FU2duYmRYdeCgxc3+p7hWbbswdIAtvw2lTUC/tIHnkvzZTwLj36
 5hw0A
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

This is the libsoup 3 version of WebKitGTK's GTK 3 API.

(cherry picked from commit be0d883331f45a8809d913043fb9da487c990a75)
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index aabaa4b895fe..3f551d8c570b 100644
--- a/meson.build
+++ b/meson.build
@@ -292,7 +292,7 @@ libvte_dep = dependency('vte-2.91', version: '>= 0.65.0')
 libxml2_dep = dependency('libxml-2.0', version: '>= 2.9.0')
 
 if get_option('webkit').enabled()
-  libwebkit_dep = dependency('webkit2gtk-4.0', version: '>= 2.26')
+  libwebkit_dep = dependency('webkit2gtk-4.1')
   config_h.set10('HAVE_WEBKIT', 1)
 endif
 
-- 
2.49.0

